coderd/users.go

```diff
@@ -867,9 +876,52 @@ func (api *API) putUserStatus(status database.UserStatus) func(rw http.ResponseW
 			})
 			return
 		}
+		httpapi.Write(ctx, rw, http.StatusOK, db2sdk.User(targetUser, organizations))
+	}
+}
+
+func (api *API) notifyUserStatusChanged(ctx context.Context, user database.User, status database.UserStatus) error {
+	var key string
+	var adminTemplateID, ownerTemplateID uuid.UUID
+	switch status {
+	case database.UserStatusSuspended:
+		key = "suspended_account_name"
+		adminTemplateID = notifications.TemplateUserAccountSuspended
+		ownerTemplateID = notifications.TemplateYourAccountSuspended
+	case database.UserStatusActive:
+		key = "activated_account_name"
+		adminTemplateID = notifications.TemplateUserAccountActivated
+		ownerTemplateID = notifications.TemplateYourAccountActivated
+	default:
+		api.Logger.Error(ctx, "user status is not supported", slog.F("username", user.Username), slog.F("user_status", string(status)))
+		return xerrors.Errorf("unable to notify admins as the user's status is unsupported")
+	}
+
+	userAdmins, err := findUserAdmins(ctx, api.Database)
+	if err != nil {
+		return xerrors.Errorf("unable to find user admins: %w", err)
```

Comment:
We should still be able to send notifications to the affected user though even if this occurs, no?